### PR TITLE
Add CLI command to get number of days remaining for 1st of next month

### LINKED ISSUE #22: Add CLI command to get number of days remaining for 1st of next month
### Description

This command is most likely handy when you want to know :
1.  what the **last day in current month** is
2. how many days is remaining for **1st of next month**

## Command
```bash
miti next
```

### Sample output

Let's say today is **Ashadh 15, 2080**. The **last day** of Ashadh in 2080 is **31** and **number of days** left for **1st Shrawan** is **17**.  The output should be something like:

> 17 days left until 1st Shrawan
Current month's last date => 31 Ashadh

### PR DESCRIPTION
**Pull request by xkshitizx**
closes #22 

### Note
This PR creates a command line option **next** which informs about days remaning for 1st of next month.

### Command
`miti next`

### Expected Output

> 17 days left until 1st [next month eg: 1st Shrawan]
> Current month's last date => [last date eg: 31 Ashadh]

### CODE REVIEW

File: lib/cli.rb [hunk 1]
```diff
@@ -63,6 +63,16 @@ def to_ad(nepali_date)
       @shell.say(output_txt, @output_color)
     end
 
+    desc "next", "get remaining days for 1st of next month and last day of current month"
+    def next
```
Comment (sanzaymdr) [inline lib/cli.rb:67]: Quick question
The command "next" is unclear as to what the `miti` achieves. It's not a big deal, we do have documentation. I'm simply curious about the command. WDYT?

Comment (xkshitizx) [reply lib/cli.rb:67]: Well, I couldn't think of better name. I agree that the name seems kind of unclear at first glance. My first thought was 'next1'. Can you think of something better?

Comment (sanzaymdr) [reply lib/cli.rb:67]: Well, I'm out of ideas too. Let's stick with this one for now, we do have documentation.

Comment (xkshitizx) [reply lib/cli.rb:67]: Cool

**Review by sanzaymdr — APPROVED** [review]

**Pull request merged**